handlers/postitem/postItem.go

```diff
@@ -0,0 +1,71 @@
+package postitem
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/mount-joy/thelist-lambda/data"
+	"github.com/mount-joy/thelist-lambda/db"
+	"github.com/mount-joy/thelist-lambda/handlers/iface"
+)
+
+type postItem struct {
+	db db.DB
+}
+
+// New returns an instance of postItem satisfying the RouteHandler interface
+func New() iface.RouteHandler {
+	return &postItem{
+		db: db.DynamoDB(),
+	}
+}
+
+// Match returns true if this RouteHandler should handle this request
+func (g *postItem) Match(request events.APIGatewayV2HTTPRequest) bool {
+	// POST /lists/<list_id>/items
+	var re = regexp.MustCompile(`^/lists/([\w-]+)/items/?$`)
+	return request.RequestContext.HTTP.Method == "POST" && re.MatchString(request.RequestContext.HTTP.Path)
+}
+
+// Handle handles this request and returns the response and status code
```

Comment:
Again, maybe these comments should be more specific. What does a post item request actually do?
`Handle posts the item in the request body to the dynamo db. The returned interface can can be converted to json........... etc.` or something